plugin/plugin.go

```diff
@@ -166,7 +172,7 @@ func Install(w *app.Window, evt event.Event) {
 			h.processFrameEvent((*unsafeOps)(unsafe.Pointer(&frame.Internal)))
 
 			for _, index := range h.redirectEvent[reflect.TypeOf(EndFrameEvent{})] {
-				h.plugins[index].ListenEvents(EndFrameEvent{})
+				h.plugins[index].ListenEvents(evt)
 			}
```

Comment:
The `EndFrameEvent{}` is send when a frame is processed. The `processFrameEvent` function is already broadcasting `FrameEvent` (evt).